CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v1.6.0 (Released 2021-01-12)
+
+BUG FIXES
+
+- fileHeader: fix ImmediateDestination formatting when BypassDestinationValidation is set
```

Comment:
I tend to wrap exact code elements (e.g. `ImmediateDestination`) in backticks. It helps to signify exact when code is specified. (CIE would not be treated as code)

Reply:
I'll update that, thanks for pointing out